compatibility/docker/kafka/Dockerfile

```diff
@@ -40,6 +40,9 @@ VOLUME /var/lib/kafka
 
 ENV advertised_host=""
 
+# ruok is not whitelisted by default in new kafka version
+ENV KAFKA_OPTS="-Dzookeeper.4lw.commands.whitelist=*"
```

Comment:
mmm interesting, however this is a Zookeeper setting not Kafka's correct?

Reply:
+1 :)

Reply:
yes, sure :) i was referring that it's different for the new version of this package (which is depends on Kafka version) - http://mirror.23media.de/apache/kafka/${kafka_version}/kafka_${scala_version}-${kafka_version}.tgz